src/index.ts

```diff
@@ -365,34 +433,41 @@ export default class MondayConnector extends BaseHttpConnector<
     `)
   }
 
-  createWebhook(
-    boardId: number,
+  async createWebhook(
+    boardId: number | string,
     url: string,
     event: string,
-    config?: Record<string, unknown>,
-  ): Promise<MondayApiReponse['data']> {
-    return this.query(CREATE_WEBHOOK_QUERY, {
-      board_id: boardId,
-      url,
-      event,
-      config,
-    })
+    columnId?: string,
+  ): Promise<string> {
+    const params: any = { board_id: Number(boardId), url, event }
+    if (event === 'ChangeSpecificColumnValue') {
+      params.config = { columnId }
+    }
+    const res = await this.query(CREATE_WEBHOOK_QUERY, params)
+    if (!res) {
+      throw new Error(`Failed to create webhook: boardId=${boardId} event=${event}`)
+    }
+    return res.create_webhook.id
   }
 
   deleteWebhook(id: number): Promise<MondayApiReponse['data']> {
     return this.query(DELETE_WEBHOOK_QUERY, { id })
   }
 
-  onRemoveEvent(event: EventConfiguration): void {
```

Comment:
don't we need that method? where do we call `deleteWebhook`?

Reply:
Right! added back